src/star/star_solo/argument_groups_solo.yaml

```diff
@@ -0,0 +1,230 @@
+argument_groups:
+- name: STARsolo (single cell RNA-seq) parameters
+  arguments:
+  - name: --soloType
+    type: string
+    description: |-
+      type of single-cell RNA-seq
+
+      - CB_UMI_Simple   ... (a.k.a. Droplet) one UMI and one Cell Barcode of fixed length in read2, e.g. Drop-seq and 10X Chromium.
+      - CB_UMI_Complex  ... multiple Cell Barcodes of varying length, one UMI of fixed length and one adapter sequence of fixed length are allowed in read2 only (e.g. inDrop, ddSeq).
+      - CB_samTagOut    ... output Cell Barcode as CR and/or CB SAm tag. No UMI counting. --readFilesIn cDNA_read1 [cDNA_read2 if paired-end] CellBarcode_read . Requires --outSAMtype BAM Unsorted [and/or SortedByCoordinate]
+      - SmartSeq        ... Smart-seq: each cell in a separate FASTQ (paired- or single-end), barcodes are corresponding read-groups, no UMI sequences, alignments deduplicated according to alignment start and end (after extending soft-clipped bases)
+    multiple: yes
+    multiple_sep: ;
+  - name: --soloCBtype
+    type: string
+    description: |-
+      cell barcode type
+
+      Sequence: cell barcode is a sequence (standard option)
+      String: cell barcode is an arbitrary string
+    example: Sequence
+  - name: --soloCBwhitelist
+    type: string
+    description: |-
+      file(s) with whitelist(s) of cell barcodes. Only --soloType CB_UMI_Complex allows more than one whitelist file.
+
+      - None            ... no whitelist: all cell barcodes are allowed
+    multiple: yes
+    multiple_sep: ;
+  - name: --soloCBstart
+    type: integer
+    description: cell barcode start base
+    example: 1
+  - name: --soloCBlen
+    type: integer
+    description: cell barcode length
+    example: 16
+  - name: --soloUMIstart
+    type: integer
+    description: UMI start base
+    example: 17
+  - name: --soloUMIlen
+    type: integer
+    description: UMI length
+    example: 10
+  - name: --soloBarcodeReadLength
+    type: integer
+    description: |-
+      length of the barcode read
+
+      - 1   ... equal to sum of soloCBlen+soloUMIlen
+      - 0   ... not defined, do not check
```

Comment:
Everywhere this syntax is used I think `choices` is warranted?

Reply:
I don't think so. In this case, you could probably pass values greater than 1, no?